spec/models/manageiq/providers/redhat/infra_manager/provision/state_machine_spec_helper.rb

```diff
@@ -0,0 +1,65 @@
+module ManageIQ::Providers::Redhat::InfraManager::Provision::StateMachineSpecHelper
+  include MiqProvision::StateMachineSpecHelper
```

Comment:
I'm confused by the name of this module and the included module...
```ruby
 ::Provision::StateMachineSpecHelper
MiqProvision::StateMachineSpecHelper
```

I can't follow what each one is since the only difference is the namespace and Miq prefix in the latter.